### PR TITLE
Add support of arithmetical operations for primitive types

### PR DESCRIPTION
**Pull request by iarovyi**
@dadhi

### CODE REVIEW

**Comment by dadhi** [comment]
Thank you, will look soon.

File: src/FastExpressionCompiler/FastExpressionCompiler.cs [hunk 1]
```diff
@@ -2269,6 +2278,57 @@ private static bool EmitComparison(BinaryExpression expr, IList<ParameterExpress
...
+                    //TODO: implement arithmetic operations for non primitive classes and structs (like Decimal).
+                    return false;
+                }
+                if (!TryEmit(leftExpr, leftExpr.NodeType, leftExpr.Type, ps, il, closure) ||
+                    !TryEmit(rightExpr, rightExpr.NodeType, rightExpr.Type, ps, il, closure))
+                    return false;
+
+                bool isUnsigned = leftExpr.Type == typeof(ushort) ||
+                                  leftExpr.Type == typeof(uint) ||
+                                  leftExpr.Type == typeof(ulong);
```
Comment (dadhi) [inline src/FastExpressionCompiler/FastExpressionCompiler.cs]: General question, should we include `byte` as well?

Comment (iarovyi) [reply src/FastExpressionCompiler/FastExpressionCompiler.cs]: Wow, good catch.
Thanks for a good comment.

**Comment by iarovyi** [comment]
Thanks for good comment.

Corrected and added specs.
@dadhi

**Comment by dadhi** [comment]
Thank you, merged!

**Pull request merged**